TVCEXECUTIVE.s

```diff
@@ -100,7 +100,7 @@ ROLLPREP	CAE	CDUX		# UPDATE ROLL LADDERS (NO NEED TO RESTART-
 		XCH	OGANOW		# 	PROTECT, SINCE ROLL DAPS RE-START)
 		XCH	OGAPAST
 		
-		CAE	OGAD		# PREPARE ROLL FDAI NEEDLE WTIH FLY-TO
+		CAE	OGAD		# PREPARE ROLL FDAI NEEDLE WITH FLY-TO
```

Comment:
I guess it's useless to argue wtih a pull-request, but I do hope you read and fix this.